x/distribution/spec/03_begin_block.md

```diff
@@ -0,0 +1,80 @@
+<!--
+order: 3
+-->
+
+# Begin Block
+
+At each `BeginBlock`, the fees received in the previous block are transferred to the distribution `ModuleAccount`, as it's the account the one who keeps track of the flow of coins in (as in this case) and out the module. The fees are also allocated to the proposer, community fund and global pool. When the validator is the proposer of the round, that validator (and their delegators) receives between 1% and 5% of fee rewards, the reserve community tax is then charged, then the remainder is distributed proportionally by voting power to all bonded validators independent of whether they voted (social distribution). Note the social distribution is applied to proposer validator in addition to the proposer reward.
+
+The amount of proposer reward is calculated from pre-commits Tendermint messages in order to incentivize validators to wait and include additional pre-commits in the block. All provision rewards are added to a provision reward pool which validator holds individually (`ValidatorDistribution.ProvisionsRewardPool`).
+
+## The Distribution Scheme
+
+See [params](07_params.md) for description of parameters.
+
+Let `fees` be the total fees collected in the previous block. All fees are
+collected in a specific module account during the block. During `BeginBlock`,
+they are sent to the `"distribution"` `ModuleAccount`. No other sending of
+tokens occur. Instead, the rewards each account is entitled to are stored, and
+withdrawals can be triggered through the messages `FundCommunityPool`,
+`WithdrawValidatorCommission` and `WithdrawDelegatorReward`.
```

Comment:
Seams we are repeating the previous paragraph. Suggestion: let's make the previous paragraph shorter and only mention there that the reward allocation happens in the Begin Block for the transactions and proposer of the previous block.

Reply:
Done! Please have a look again and tell me what you think